packages/css-library/src/stylesheets/elements.scss

```diff
@@ -5,3 +5,197 @@
 // p {
 //    @include @usa-paragraph
 // }
+
+@use "../../dist/tokens/scss/variables" as *;
+
+$color-link-default-hover: rgba($color-black, 0.05);
```

Comment:
This color token is not available yet in the CSS Library. I'm adding it to the stylesheet directly for now because I'm not sure yet how to get the rgba value to work with the token build.

This token is being used in several different places in Formation though so we will want to get it added more globally somehow.

<img width="792" alt="Screenshot 2023-12-11 at 2 42 40 PM" src="https://github.com/department-of-veterans-affairs/component-library/assets/872479/461804e2-7dd5-432a-b157-63c69a56d832">

Reply:
resolved here https://github.com/department-of-veterans-affairs/component-library/pull/982#discussion_r1424480123